Resources/views/Form/karser_recaptcha3_widget.html.twig

```diff
@@ -4,14 +4,17 @@
         {{ block('form_widget_simple') }}
 
         {% if form.vars.enabled -%}
-            <script src="https://www.google.com/recaptcha/api.js?render={{ form.vars.site_key }}"></script>
-            <script>
-                grecaptcha.ready(function() {
-                    grecaptcha.execute('{{ form.vars.site_key }}', {action: '{{ form.vars.action_name }}'}).then(function(token) {
-                        document.getElementById('{{ id }}').value = token;
+            <script type="text/javascript">
```

Comment:
`type="text/javascript"` is redundant

Reply:
Right! I copy-pasted this callback function from Google's docs and didn't even notice that ;)